blocks/init/src/blocks/components/video/video.php

```diff
@@ -10,23 +10,17 @@
 
 $media = $attributes['media'] ?? [];
 
-if ( empty( $media['url'] ) ) {
+if ( ! $media['url'] ) {
```

Comment:
won't this throw a warning if `$media['url']` key is undefined?

Reply:
tnx